docs/jwst/persistence/reference_files.rst

```diff
@@ -1,33 +1,157 @@
+DQ_DEF   BINTABLE    2    TFIELDS = 4     N/A
+=======  ========  =====  ==============  =========
+
+.. include:: ../includes/dq_def.rst
+
+.. _TRAPPARS:
+
+TRAPPARS Reference File
+-----------------------
+
+:REFTYPE: TRAPPARS
+:Data model: `~jwst.datamodels.TrapParsModel`
+
+The TRAPPARS reference file contains default parameter values
+used in the persistence correction.
+
 .. include:: trappars_selection.rst
 
-At the present time, there are no reference files for MIRI, and CRDS
-will return "N/A" for the names of the files if the persistence step
-is run on MIRI data, in which case the input will be returned unchanged
-except that the primary header keyword S_PERSIS will will have been
-set to 'SKIPPED'.
+.. include:: ../includes/standard_keywords.rst
+
+Type Specific Keywords for TRAPPARS
++++++++++++++++++++++++++++++++++++
+In addition to the standard reference file keywords listed above,
+the following keywords are *required* in TRAPPARS reference files,
+because they are used as CRDS selectors
+(see :ref:`trappars_selectors`):
 
-Reference File Formats
-----------------------
-The TRAPDENSITY reference file contains an IMAGE extension that gives
-the density of traps at each pixel.
+=========  ==============================
+Keyword    Data Model Name
+=========  ==============================
+DETECTOR   model.meta.instrument.detector
+=========  ==============================
 
-The PERSAT reference file contains an IMAGE extension that gives the
-persistence saturation threshold (full well) at each pixel.
+Reference File Format
++++++++++++++++++++++
+TRAPPARS reference files are FITS format, with 1 BINTABLE extension.
+The FITS primary HDU does not contain a data array.
+The format and content of the file is as follows:
 
-The TRAPPARS reference file contains a BINTABLE extension with four
-float (double precision) columns:
+========  ========  ===========
+EXTNAME   XTENSION  Dimensions
+========  ========  ===========
+TRAPPARS  BINTABLE  TFIELDS = 4
+========  ========  ===========
 
-* capture0: the coefficient of the exponential capture term
-* capture1: minus the reciprocal of the capture e-folding time
-* capture2: the "instantaneous" capture coefficient
-* decay_param: minus the reciprocal of the decay e-folding time
+The format and contents of the table extension is as follows:
+
++--------------+-----------+------------------------------------------------+
+| Column name  | Data type | Description                                    |
++==============+===========+================================================+
+| capture0     | float     | Coefficient of exponential capture term        |
++--------------+-----------+------------------------------------------------+
+| capture1     | float     | Minus the reciprocal of capture e-folding time |
++--------------+-----------+------------------------------------------------+
+| capture2     | float     | The "instaneous" capture coefficient           |
```

Comment:
Typo:  "instaneous" should be "instantaneous".

Reply:
👍 Fixed in updated commit.